website/blog/2025-09-15-make-judge/index.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: "Beyond Generic Metrics: Build Domain-Specific LLM Judges with MLflow"
+description: How to create custom evaluators that actually understand your quality requirements
+slug: custom-llm-judges-make-judge
+authors: [mlflow-maintainers]
+tags: [genai, evaluation, judges, llm]
+thumbnail: /img/blog/make-judge-thumbnail.png
+image: /img/blog/make-judge-thumbnail.png
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+In this post, we'll show how MLflow Scorers—the framework's core abstraction for automated evaluation—make it simple to build judges that actually understand your domain-specific quality requirements.
+
+[MLflow Scorers](https://mlflow.org/docs/latest/genai/eval-monitor/) are evaluation functions that can assess any aspect of your GenAI application. The `make_judge` API lets you create these scorers using natural language instructions, and here's the key insight: whether you're using MLflow's built-in scorers or creating custom ones, they all share the same powerful capabilities.
+
+If you've tried evaluating GenAI applications, you know generic metrics miss what matters. Your customer support bot needs to be evaluated on empathy and problem resolution. Your code generator needs to be checked for security vulnerabilities. Your medical advisor needs domain-specific accuracy checks. MLflow Scorers let you define these requirements in plain English.
+
+This post demonstrates the power of MLflow Scorers by showing how to:
+
+- Create custom scorers with `make_judge` using natural language instructions
+- Build scorers that can act agentically, analyzing entire execution traces
+- Align scorers with human feedback to improve their accuracy over time
+- Share and version scorers across your team
+
+We'll build a customer support quality scorer, show how it can evaluate complex agent behaviors, and demonstrate how human feedback makes it smarter. The built-in scorers MLflow provides? They're just pre-defined versions using these same capabilities. Once you understand this, you'll see how flexible and powerful MLflow's evaluation framework really is.
```

Comment:
```suggestion
To illustrate the power of the new API, we will build a customer support quality scorer, show how it can evaluate complex agent behaviors, and demonstrate how human feedback makes it more effective. The built-in scorers MLflow provides? They're just pre-defined versions using these same capabilities. Once you understand this, you'll see how flexible and powerful MLflow's evaluation framework really is.
```